cpp/src/io/parquet/writer_impl.cu

```diff
@@ -85,6 +87,31 @@ parquet::Compression to_parquet_compression(compression_type compression)
   }
 }
 
+size_type column_size(column_view const& column, rmm::cuda_stream_view stream)
+{
+  if (column.num_children() == 0) {
+    return is_fixed_width(column.type()) ? size_of(column.type()) * column.size() : 0;
+  }
+
```

Comment:
Since this is passed leaf columns, do I need to check for STRUCT at all?  And does this work for binary columns (list[uint_8] right?)?